build.sh

```diff
@@ -40,34 +62,51 @@ pushd $WEBRTC_SRC
 git checkout $COMMIT
 popd
 
-echo "Generating build scripts ..."
-pushd $WEBRTC_SRC
-python webrtc/build/gyp_webrtc
+echo "Cleaning webrtc ..."
+pushd $WEBRTC_SRC || exit 1
+rm -rf out/$CONFIG
+popd
+
+echo "Applying webrtc patches ..."
+pushd $WEBRTC_SRC || exit 1
+for PATCH in build_at_webrtc_branch_heads_52.patch; do
+	git apply --check ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
+	git am < ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
+done
 popd
 
 echo "Building webrtc ..."
 pushd $WEBRTC_SRC
-ninja -C out/$CONFIG
+export GYP_DEFINES="include_tests=0 include_examples=0"
+python webrtc/build/gyp_webrtc webrtc/api/api.gyp || exit 1
```

Comment:
Ah right, I forgot this change results in the `Undefined symbols for ... webrtc::field_trial::FindFullName`.  Sorry :(